generate_attestation_file.sh

```diff
@@ -0,0 +1,142 @@
+          "frequency": "weekly",
+          "explanation": "Non-expired Status passed"
+        },
+        {
+          "control_id": "test-control-7",
+          "status": "passed",
+          "updated": "'"$(date -I)"'",
+          "updated_by": "John Doe, ISSO",
+          "frequency": "every3days",
+          "explanation": "Non-expired Status passed"
+        },
+        {
+          "control_id": "test-control-8",
+          "status": "passed",
+          "updated": "'"$(date -I)"'",
+          "updated_by": "John Doe, ISSO",
+          "frequency": "daily",
+          "explanation": "Non-expired Status passed"
+        },
+        {
+          "control_id": "test-control-9",
+          "status": "passed",
+          "updated": "'"$(date -d "-13140:00:00"  -I)"'",
+          "updated_by": "John Doe, ISSO",
+          "frequency": "annually",
+          "explanation": "Expired Status passed"
+        },
+        {
+          "control_id": "test-control-10",
+          "status": "passed",
+          "updated": "'"$(date -d "-6570:00:00"  -I)"'",
+          "updated_by": "John Doe, ISSO",
+          "frequency": "semiannually",
+          "explanation": "Expired Status passed"
+        },
+        {
+          "control_id": "test-control-11",
+          "status": "passed",
+          "updated": "'"$(date -d "-4380:00:00"  -I)"'",
+          "updated_by": "John Doe, ISSO",
+          "frequency": "quarterly",
+          "explanation": "Expired Status passed"
+        },
+        {
+          "control_id": "test-control-12",
+          "status": "passed",
+          "updated": "'"$(date -d "-1095:00:00"  -I)"'",
+          "updated_by": "John Doe, ISSO",
+          "frequency": "monthly",
+          "explanation": "Expired Status passed"
+        },
+        {
+          "control_id": "test-control-13",
+          "status": "passed",
+          "updated": "'"$(date -d "-504:00:00"  -I)"'",
+          "updated_by": "John Doe, ISSO",
+          "frequency": "every2weeks",
+          "explanation": "Expired Status passed"
+        },
+        {
+          "control_id": "test-control-14",
+          "status": "passed",
+          "updated": "'"$(date -d "-252:00:00"  -I)"'",
+          "updated_by": "John Doe, ISSO",
+          "frequency": "weekly",
+          "explanation": "Expired Status passed"
+        },
+        {
+          "control_id": "test-control-15",
+          "status": "passed",
+          "updated": "'"$(date -d "-96:00:00"  -I)"'",
+          "updated_by": "John Doe, ISSO",
+          "frequency": "every3days",
+          "explanation": "Expired Status passed"
+        },
+        {
+          "control_id": "test-control-16",
+          "status": "passed",
+          "updated": "'"$(date -d "-25:00:00"  -I)"'",
+          "updated_by": "John Doe, ISSO",
+          "frequency": "daily",
+          "explanation": "Expired Status passed"
+        }
+      ]
+    }
+  }
+}
+'
+echo $JSON_STRING > attestations.json
```

Comment:
It makes a bit more sense here to just print to the screen so that the user can redirect the output into a file afterwards if they so choose.

Reply:
oh no this is for the automated tests. this script generates attestation file with timestamp relative to run date, coz expiration is being tested.

i should move this to the test folder